db/migrate/20170824103029_add_timestamps_to_status_pins.rb

```diff
@@ -0,0 +1,5 @@
+class AddTimestampsToStatusPins < ActiveRecord::Migration[5.1]
+  def change
+    add_timestamps :status_pins, null: true
```

Comment:
The feature is not included in any release yet and we do not have to maintain the compatibility. What about truncate the table and set `null: false`?